test/distributed/fsdp/test_fsdp_comm_hooks.py

```diff
@@ -34,7 +35,7 @@
     torch.cuda.is_available()
     and torch.version.cuda is not None
     and int(torch.version.cuda.split(".")[0]) >= 11
-)
+) or TEST_WITH_ROCM
```

Comment:
@lcskrishna I think a better construct would be the following?
```suggestion
BFLOAT16_AVAILABLE = (
    torch.cuda.is_available()
    and ((torch.version.cuda is not None
    and int(torch.version.cuda.split(".")[0]) >= 11)
    or torch.version.hip is not None)
)
```